### ISSUE TITLE
The UX for the gear icon isn't great in iOS 8

### ISSUE THREAD

**Issue by stubailo** [percolatestudio/mdg-todos#22]
In iOS 8, input type="select" doesn't work quite the same way as before; in the new UI, it doesn't make as much sense to represent a dropdown menu with a select element, because you have to scroll to a certain item and then click "done".

The gear icon dropdown should probably be replaced with an HTML dropdown, or several icons.

**Comment by tmeasday** [comment]
I think this was actually the way it behaved in iOS7 too, no? Over to you @dominicnguyen

**Comment by stubailo** [comment]
It's a little nicer in Android, I think, but it's definitely not great in iOS.

**Comment by domyen** [comment]
@stubailo This was the expected behavior in iOS. The reasons to use the  <select> in this case are twofold:
1. Simplicity: We use a standard HTML element whose interaction is supported by out-of-the-box by mobile browsers instead of adding more code complexity to power a more custom UI. 
2. Affordances for more list items in the future

That said, we're happy to add a custom UI that is more similar to native mobile patterns than mobile-web patterns; just say the word :D.

**Comment by stubailo** [comment]
Yeah, I think we should replace it with out own UI. Normally, I'm in favor of this design. I used to use it all the time, so I totally get where you are coming from. In this case, however, I don't think it's really working out.

Maybe we can add another sidebar on the right for list options? If we added sharing private lists with other people those settings could go in the new sidebar as well.

**Comment by domyen** [comment]
From the in-person meeting:
We'll create a custom UI that lives alongside the sharing feature. It will likely be a page overlay that appears when the cog icon is tapped and serve as a sort of control panel for the list.

**Comment by domyen** [comment]
Attached a design to address the <select > UX with a more native-like flow. @stubailo check it out? 

**Notes:**
1. Accommodate for being able to explicitly share private lists (public lists are automatically sharable and thus don't need to see sharing UI)
2. Keep the number of distinct templates for web and mobile to a minimum

![screenshot 2014-09-29 09 22 25](https://cloud.githubusercontent.com/assets/263385/4444915/dba0ac3c-47f4-11e4-8eb6-d264850ea526.png)

p.s., the control panel idea has edge cases that make it less flexible to our needs than expected

**Comment by zol** [comment]
@stubailo - check out the proposed design. Our plan is to replace the UX with this design as soon as we can (won't make it into today's release).

**Comment by stubailo** [comment]
Does it make sense for the bar to have a share button if the list isn't private? What would it do?

**Comment by domyen** [comment]
@stubailo Since lists default to public, we would not show the share button for public lists.

**Comment by stubailo** [comment]
The mockup looks good for mobile!

**Comment by stubailo** [comment]
Any updates on this new UI/feature?

**Comment by zol** [comment]
@stubailo - we were planning on holding off on this along with the private sharing feature till after Sous is released as Matt is keen to push Sous forward along with the new recipe tweeting feature. Is that cool or do we need to re-adjust priorities?

**Comment by stubailo** [comment]
@debergalis what do you think?

**Comment by debergalis** [comment]
I think that's right.
